[v38]  rk3399 platform image fails to enter ES without keyboard

This phenomenon exists in versions 36 37 38

This is part of /userdata/system/logs/sway.log

```
00:00:00.195 [INFO] [sway/commands.c:380] Config command: exec /usr/bin/emulationstation-standalone
00:00:00.195 [INFO] [sway/commands.c:400] After replacement: exec /usr/bin/emulationstation-standalone
00:00:00.195 [DEBUG] [sway/config.c:876] Deferring command `exec /usr/bin/emulationstation-standalone'
00:00:00.195 [ERROR] [sway/config.c:433] Error(s) loading config!
sh: 1: swaynag: not found
00:00:00.243 [INFO] [sway/server.c:299] Starting backend on wayland display 'wayland-1'
00:00:00.243 [DEBUG] [wlr] [backend/libinput/backend.c:87] Starting libinput backend
00:00:00.260 [ERROR] [wlr] [backend/libinput/backend.c:111] libinput initialization failed, no input devices
00:00:00.260 [ERROR] [wlr] [backend/libinput/backend.c:112] Set WLR_LIBINPUT_NO_DEVICES=1 to suppress this check
00:00:00.260 [ERROR] [wlr] [backend/multi/backend.c:33] Failed to initialize backend.
00:00:00.260 [ERROR] [sway/server.c:302] Failed to start backend
00:00:00.261 [DEBUG] [wlr] [types/wlr_drm_lease_v1.c:103] Destroying wlr_drm_lease_device_v1 for /dev/dri/card0
00:00:00.261 [INFO] [sway/main.c:419] Shutting down sway
2013-01-18 09:52:24 - [main.c:582] wl_display_roundtrip failed
00:00:00.333 [DEBUG] [wlr] [types/wlr_drm_lease_v1.c:685] Destroying wlr_drm_lease_v1_manager
[root@BATOCERA /userdata/system]# 
```

There is no keyboard detected by libinput, causing sway to refuse to start. Setting WLR_LIBINPUT_NO_DEVICES=1 as mentioned in the log will work.

```
diff --git a/package/batocera/emulationstation/batocera-emulationstation/wayland/sway-launch b/package/batocera/emulationstation/batocera-emulationstation/wayland/sway-launch
index 105281f75b..679ec7d9b7 100755
--- a/package/batocera/emulationstation/batocera-emulationstation/wayland/sway-launch
+++ b/package/batocera/emulationstation/batocera-emulationstation/wayland/sway-launch
@@ -1,4 +1,4 @@
 #!/bin/sh
 
 SWAY_LOG_FILE=/userdata/system/logs/sway.log
-/usr/bin/sway -d > ${SWAY_LOG_FILE} 2>&1
+WLR_LIBINPUT_NO_DEVICES=1 /usr/bin/sway -d > ${SWAY_LOG_FILE} 2>&1
```

There should be other solutions? Because x86 images don’t have this problem

--- a/package/batocera/emulationstation/batocera-emulationstation/wayland/sway-launch
+++ b/package/batocera/emulationstation/batocera-emulationstation/wayland/sway-launch
@@ -1,4 +1,4 @@
 #!/bin/sh
 
 SWAY_LOG_FILE=/userdata/system/logs/sway.log
-/usr/bin/sway -d > ${SWAY_LOG_FILE} 2>&1
+WLR_LIBINPUT_NO_DEVICES=1 /usr/bin/sway -d > ${SWAY_LOG_FILE} 2>&1

